### PR TITLE
[WIP] add plugins to docs

### PR DESCRIPTION
**Pull request by DavidWells**
## What did you implement:

***Implementing Issue:*** #12345

Adding list of available plugins to the docs.

Additions welcome =)

### CODE REVIEW

**Comment by pmuens** [comment]
@DavidWells thanks for the list!

Here are 2 additions:

- https://github.com/nfour/serverless-build-plugin
- https://github.com/wei-xu-myob/serverless-scriptable-plugin

taken from https://github.com/JustServerless/awesome-serverless/blob/master/README.md#plugins

File: docs/01-guide/09-installing-plugins.md [hunk 1]
```diff
@@ -53,4 +53,12 @@ plugins:
...
 In this case `plugin1` is loaded before `plugin2`.
 
+
+# List of available plugins
+
+* [serverless-webpack](https://github.com/elastic-coders/serverless-webpack) - bundle your lambdas with Webpack
+* [serverless-alexa-plugin](https://github.com/rajington/serverless-alexa-plugin) - support Alexa Lambda events
+* [serverless-plugin-write-env-vars](https://github.com/silvermine/serverless-plugin-write-env-vars)
+* [serverless-plugin-multiple-responses](https://github.com/silvermine/serverless-plugin-multiple-responses)
+
```
Comment (pmuens) [inline docs/01-guide/09-installing-plugins.md]: Could we add a description for the last two plugins as well?

Comment (DavidWells) [inline docs/01-guide/09-installing-plugins.md]: Is https://github.com/silvermine/serverless-plugin-write-env-vars solved/obsolete with new variable setup?

Comment (pmuens) [inline docs/01-guide/09-installing-plugins.md]: Write env vars will create a file with variables you can then access in your lambda function. Our variable system doesn't cover the environment variable access in Lambda yet.

**Comment by andymac4182** [comment]
Should this be included in here or the https://github.com/serverless/community-plugins repo? There will be a lot of plugins related just to various providers and languages soon. A page dedicated to showing these / site like http://library.octopusdeploy.com/listing would be great to allow people to navigate around.

File: lib/classes/CLI.js [hunk 2]
```diff
@@ -67,7 +67,7 @@ class CLI {
     this.consoleLog('');
 
     this.consoleLog(chalk.yellow.underline('Commands'));
-    this.consoleLog(chalk.dim('* Serverless documentation: http://docs.serverless.com'));
```
Comment (flomotlik) [inline lib/classes/CLI.js]: Please remove changing of the docs from here for now. Opened a new issue in #2167 to be able to collect some feedback from the community before we release in the next few days.

File: docs/01-guide/09-installing-plugins.md (hunk 1 above)
Comment (svdgraaf) [inline docs/01-guide/09-installing-plugins.md]: @DavidWells you can add this one as well: https://github.com/svdgraaf/serverless-plugin-stage-variables 😄

**Comment by eahefnawy** [comment]
Outdated...closing...

**Pull request closed without merging**